docs/source/usage/plugins/openPMD.rst

```diff
@@ -43,9 +43,9 @@ The openPMD API will parse the file name to decide the chosen backend and iterat
 In order to set defaults for these value, two further options control the filename:
 
 * ``--openPMD.ext`` sets the filename extension.
-  Possible extensions include ``bp`` for the ADIOS2 backends (default), ``h5`` for HDF5 and ``sst`` for Streaming via ADIOS2/SST.
-  If the openPMD API has been built with support for the ADIOS1 and ADIOS2 backends, ADIOS2 will take precedence over ADIOS1.
-  This behavior can be overridden by setting the environment variable ``OPENPMD_BP_BACKEND=ADIOS1``.
+  Possible extensions include ``bp`` for the ADIOS2 backend (default), ``h5`` for HDF5 and ``sst`` for Streaming via ADIOS2/SST.
+  If the openPMD API has been compiled with support for ADIOS2, the openPMD API will automatically prefer using ADIOS2 over ADIOS1.
```

Comment:
If I understand @franzpoeschel [last comment](https://github.com/ComputationalRadiationPhysics/picongpu/pull/3795#discussion_r710983399) corretly, using ADIOS(1) no option anymore.
Shouldn't than this sentence state:
```suggestion
  If the openPMD API has been compiled with support for ADIOS2, the openPMD API will automatically prefer using ADIOS2 over HDF5.
```

Reply:
Those two sentences should go entirely, good catch. There is no preference of ADIOS2 over HDF5 or the other way around in openpMD-api, this was just there because the `.bp` extension could be used to select ADIOS2 as well as ADIOS1.

Reply:
I think the sensence is not about that. But it (plus the next one) should warn users who has both ADIOS1 and ADIOS2 in their openPMD, that in order to use it with PIConGPU they should take care of that enrironment variable

Reply:
True, you can leave it in for that purpose. Then, I would formulate this a bit more explicitly to warn users against selecting ADIOS1 via that variable.

Reply:
Okay, i will reformulate

Reply:
[This error message](https://github.com/ComputationalRadiationPhysics/picongpu/blob/d7a4863e9a421fda8d50df31771c83d515b64f62/include/picongpu/plugins/openPMD/openPMDWriter.hpp#L152) may be a starting point

Reply:
Fixed